### PR TITLE
super-user-spark: init at 0.1.0.0

### PR DESCRIPTION
**Pull request by badmutex**
Super-User-Spark is a program for managing dotfiles.
The author is Tom Sydney Kerckhove @NorfairKing

Details on usage can be found on his [blogpost](http://cs-syd.eu/posts/2015-07-19-super-user-spark.html) as well as the [reddit thread](http://www.reddit.com/r/haskell/comments/3dsdi5/more_than_just_another_symlink_manager_written_in/).

### CODE REVIEW

**Comment by devhell** [comment]
Hi @badi and welcome!

Thanks for your contribution, however, I would just like to point you out to our [contribution guide](https://hydra.nixos.org/build/23816503/download/1/nixpkgs/manual.html#idm140737319322080), specifically under `Format the commit in the following way:` you'll find the correct naming convention for new submissions.

In your case it would be `init`.

Thanks!

**Comment by badi** [comment]
@devhell thanks for the feedback. I've renamed the PR to conform.

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -3006,6 +3006,8 @@ let
 
   super = callPackage ../tools/security/super { };
 
+  superUserSpark = haskellPackages.callPackage ../applications/misc/super_user_spark { };
```
Comment (jagajaga) [inline pkgs/top-level/all-packages.nix]: Please rename to `super-user-spark`. We prefer to use dashes in the attribute names now too :)

**Comment by vcunat** [comment]
Looks good and an interesting tool; pushed.

**Pull request closed without merging**